garden_ai/garden.py

```diff
@@ -175,3 +212,26 @@ def register_metadata(self, garden: Garden, out_dir=None):
         else:
             with open(out_dir, "w+") as f:
                 f.write(garden.json())
+
+    def publish_garden(self, garden_id=None, garden_doc=None, visibility='Public'):
```

Comment:
is `garden_doc` supposed to be the metadata dict for the garden object? if so I think it might make more sense to have this just take a `garden` and use `garden.uuid` and `dict(garden.json())` to get the garden id and metadata dict respectively

Reply:
Yeah, when I first wrote this, the garden object didn't exist yet.  I totally agree that it should just take a garden--I'll make the change.